spring-integration-groovy/src/main/java/org/springframework/integration/groovy/GroovyScriptExecutingMessageProcessor.java

```diff
@@ -137,7 +137,7 @@ public void setCompilerConfiguration(CompilerConfiguration compilerConfiguration
 	}
 
 	@Override
-	protected ScriptSource getScriptSource(Message<?> message) {
+	protected ScriptSource getScriptSource(@Nullable Message<?> message) {
```

Comment:
Let's fix `ScriptExecutingMessageSource` instead providing a fake message instance!

The `MessageProcessor` cotnract is to not accept null for the `message` argument.

That could be a `static final` property in the `ScriptExecutingMessageSource` with `byte[0]` as payload.